src/Tizen.NUI/src/public/Common/Rotation.cs

```diff
@@ -48,6 +48,17 @@ public Rotation(Radian angle, Vector3 axis) : this(Interop.Rotation.NewRotation(
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// The constructor from two vector.
+        /// </summary>
+        /// <param name="v0">Thefirst vector.</param>
+        /// <param name="v1">The second vector.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Rotation(Vector3 v0, Vector3 v1) : this(Interop.Rotation.NewRotation2(Vector3.getCPtr(v0), Vector3.getCPtr(v1)), true)
```

Comment:
Please give detail information about v0, v1

Reply:
Thanks I added some more information about this constructor.